test/new-e2e/tests/installer/host/host.go

```diff
@@ -154,6 +162,18 @@ func (h *Host) InstallerVersion() string {
 	return strings.TrimSpace(h.remote.MustExecute("sudo datadog-installer version"))
 }
 
+// AgentVersion returns the version of the agent on the host.
+func (h *Host) AgentVersion() string {
+	rawVersion := h.remote.MustExecute("sudo datadog-agent version")
```

Comment:
Maybe not in this PR but as we discussed this will always hit the stable agent so it's not super suitable for the tests.